src/test/java/org/opensearch/knn/index/LuceneEngineIT.java

```diff
@@ -406,4 +426,12 @@ private void validateQueries(SpaceType spaceType, String fieldName) throws Excep
             }
         }
     }
+
+    private void validateQueryResultsQty(float[] searchVector, int k) throws Exception {
+        final Response response = searchKNNIndex(INDEX_NAME, new KNNQueryBuilder(FIELD_NAME, searchVector, k), k);
```

Comment:
Why `response` is final but not `responseBody`?
Also how about parameters `searchVector` and `k`?

Reply:
ack, will make all vars final